apps/test-app/frontend/src/components/app/App.tsx

```diff
@@ -386,3 +385,8 @@ function createInstanceKeysSelectQuery(keys: InstanceKey[]): PresentationQuery {
   });
   return { query, bindings };
 }
+
+function parseFullClassName(fullClassName: string) {
```

Comment:
Can't you use `parseFullClassName` from the shared package here?

Reply:
I could, but I'm about to remove all this selection-related code from test app, including the usage of this function, so didn't want to introduce the `@itwin/presentation-shared` dependency unnecessarily.